### PR TITLE
A Mound of Typo Fixes

### PR DESCRIPTION
**Pull request by detahramet**
#### Summary
None

#### Purpose of change
As Issue #68497 states, there is a mound of typos, a solid chunk of which are mine.

#### Describe the solution
I have corrected the typos as per the afforementioned sadenar.

#### Describe alternatives you've considered
Being lazy and letting other people fix up my mess.

#### Testing
N/A

#### Additional context
Yeah, my bad on the typos.

### CODE REVIEW

File: data/json/furniture_and_terrain/furniture-terrains.json [hunk 1]
```diff
@@ -1679,7 +1679,7 @@
     "move_cost_mod": -1,
     "examine_action": "chainfence",
     "coverage": 100,
-    "description": "An area of gravel heaped well above your head, difficult to climb",
+    "description": "An area of gravel heaped well above your head, difficult to climb over",
```
Comment (Qrox) [inline data/json/furniture_and_terrain/furniture-terrains.json:1682]:
```suggestion
    "description": "An area of gravel heaped well above your head, difficult to climb over.",
```

File: data/mods/Xedra_Evolved/mutations/paraclesians/salamander_mutations.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "points": 2,
     "visibility": 4,
     "ugliness": 3,
-    "description": "Your skin has has magickal flames flaring out of it on occasion.  You can operate in warmer climates than normal humans but begin to take damage from getting wet.",
+    "description": "Your skin has magickal flames flaring out of it on occasion. You can operate in warmer climates than normal humans but begin to take damage from getting wet.",
```
Comment (Qrox) [inline data/mods/Xedra_Evolved/mutations/paraclesians/salamander_mutations.json:21]:
```suggestion
    "description": "Your skin has magical flames flaring out of it on occasion.  You can operate in warmer climates than normal humans but begin to take damage from getting wet.",
```

Comment (sadenar) [reply data/mods/Xedra_Evolved/mutations/paraclesians/salamander_mutations.json:21]: This one I'm pretty sure is intended spelling within XE for otherworldly magical nonsense, but @GuardianDll might know better, if it is not intended there is quite a lot more to fix.

File: data/mods/Xedra_Evolved/items/armor/integrated.json [hunk 3]
```diff
@@ -266,7 +266,7 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str_sp": "burning coals skin" },
-    "description": "Your skin has has magickal flames flaring out of it on occasion.  You can operate in warmer climates than normal humans but begin to take damage from getting wet.",
+    "description": "Your skin has magickal flames flaring out of it on occasion. You can operate in warmer climates than normal humans but begin to take damage from getting wet.",
```
[the remark this answers is not in the document]
Comment (sadenar) [reply data/mods/Xedra_Evolved/items/armor/integrated.json:269]: Same as below for magickal.

**Comment by sadenar** [comment]
Thanks for taking this on, really appreciate the work.

File: data/json/items/armor/legs_prosthetics.json [hunk 4]
```diff
@@ -2,8 +2,8 @@
   {
     "id": "right_leg_prosthetic",
     "type": "ARMOR",
-    "name": { "str_sp": "prosthetic right leg" },
-    "description": "A prosthetic leg, used as a replacement limb for those missing one.  Its modern design and sleek ergonomics make it very comfortable and highly suitable for use, and advanced makes it look exactly like the original.  This one is for the right leg.",
+    "name": { "str": "prosthetic right leg", "str_pl": "prosthetic right legs" },
```
Comment (Maleclypse) [inline data/json/items/armor/legs_prosthetics.json:5]:
```suggestion
    "name": { "str": "prosthetic right leg" },
```

File: data/mods/Xedra_Evolved/mutations/paraclesians/salamander_mutations.json (hunk 2 above)
Comment (Maleclypse) [reply data/mods/Xedra_Evolved/mutations/paraclesians/salamander_mutations.json:21]: I only spell magick with a k.  It's a choice.    The inspirational quote that led to this state of affairs was "Never trust anyone who spells magick with a 'k'. "

**Comment by detahramet** [comment]
Sorry this has been stalled, I've been tied up with my weekend.

**Pull request merged**